reference/configuration/framework.rst

```diff
@@ -417,33 +417,46 @@ Whether to enable the ``serializer`` service or not in the service container.
 
 For more details, see :doc:`/cookbook/serializer`.
 
-templating
-~~~~~~~~~~
+assets
```

Comment:
I think we should add a label for the old `templating` key to help people finding the new options.